content/docs/hooks-reference.md

```diff
@@ -60,8 +60,8 @@ function Counter({initialCount}) {
     <>
       Count: {count}
       <button onClick={() => setCount(initialCount)}>Reset</button>
-      <button onClick={() => setCount(prevCount => prevCount - 1)}>-</button>
-      <button onClick={() => setCount(prevCount => prevCount + 1)}>+</button>
+      <button onClick={(prevCount) => setCount(prevCount => prevCount - 1)}>-</button>
+      <button onClick={(prevCount) => setCount(prevCount => prevCount + 1)}>+</button>
```

Comment:
That wouldn't work. The first argument of event callbacks is the `event` itself. `prevCount` is passed by React itself when calling functional updates. That is we wrote `setCount(prevCount => prevCount + 1)` and not `setCount(prevCount + 1)`.

Reply:
Yes, i didnt see it.